wormhole-connect/src/views/WalletModal.tsx

```diff
@@ -53,7 +53,7 @@ const useStyles = makeStyles()((theme) => ({
 
 const getReady = (wallet: Wallet) => {
   const ready = wallet.getWalletState();
-  return ready !== WalletState.Unsupported;
+  return ready !== WalletState.Unsupported && ready !== WalletState.NotDetected;
```

Comment:
If I remember correctly, sometimes this caused Solflare and Phantom to not show up in the list.  I think if you weren't signed in they wouldn't show up

Reply:
There are 4 states: `Installed`, `Loadable`, `NotDetected`, `Unsupported`. There's an additional layer beyond installed/not installed; for example: browser wallets would not be supported in other environments other than a browser, so they would yield Unsupported instead of NotDetected.